debian/templates

```diff
@@ -2,7 +2,8 @@ Template: matrix-synapse/server-name
 Type: string
 _Description: Name of the server:
  The name that this homeserver will appear as, to clients and other
- servers via federation. This name should match the SRV record
+ servers via federation. This name should match the "m.server" entry
```

Comment:
also, this implies it's mandatory to have a `.well-known` and/or a SRV record. I think we want to say something like, "Normally, this should be the public name of this server, but if you set up delegation, it can be a different domain..."

we can also link to https://github.com/matrix-org/synapse/blob/master/docs/delegate.md.